fuse/nodefs/api.go

```diff
@@ -102,6 +102,11 @@ type Node interface {
 	SetXAttr(attr string, data []byte, flags int, context *fuse.Context) fuse.Status
 	ListXAttr(context *fuse.Context) (attrs []string, code fuse.Status)
 
+	// File locking
+	GetLk(file File, owner uint64, lk *fuse.FileLock, flags uint32, padding uint32, out *fuse.FileLock, context *fuse.Context) (code fuse.Status)
```

Comment:
padding should never be part of function signatures.

Reply:
Fixed